tools/rum-models-generator/Sources/CodeDecoration/RUMCodeDecorator.swift

```diff
@@ -20,6 +20,7 @@ public class RUMCodeDecorator: SwiftCodeDecorator {
                 "RUMMethod",
                 "RUMEventAttributes",
                 "RUMCITest",
+                "RUMSyntheticsTest",
```

Comment:
```suggestion
                "RUMSessionType",
                "RUMSyntheticsTest",
```


/suggestion It will simplify quite a bit if you also set `RUMSessionType` as a shared type. You will then be able to set the session type in `RUMScopeDependencies` only once and re-use it everywhere.

Reply:
Oh that's a good point :)